ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsPaymentMethodDetails.kt

```diff
@@ -0,0 +1,29 @@
+package com.braintreepayments.api
+
+import org.json.JSONObject
+
+/**
+ * Details of a specific payment method.
+ *
+ * @property canBeVaulted Indicates if the payment method can be saved for future transactions.
+ * @property eligibleInPaypalNetwork Indicates if the payment method is eligible in the PayPal network.
+ * @property recommended Indicates if this payment method is recommended for the shopper.
+ * @property recommendedPriority The priority ranking of this payment method if recommended; null if not applicable.
+ */
+internal data class ShopperInsightsPaymentMethodDetails(
+    val canBeVaulted: Boolean?,
+    val eligibleInPaypalNetwork: Boolean?,
+    val recommended: Boolean?,
+    val recommendedPriority: Int? = null
```

Comment:
actually I am going to just remove default being null and make them non optionals since the fromJson() method creates the data object with default values anyways. ie. false and 0